File: clients/dotnet/enums/BatchType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// 
    /// </summary>
    public enum BatchType
    {
        /// <summary>
        /// 
        /// </summary>
        AvaCertUpdate,

        /// <summary>
        /// 
        /// </summary>
        AvaCertUpdateAll,

        /// <summary>
        /// 
        /// </summary>
        BatchMaintenance,

        /// <summary>
        /// 
        /// </summary>
        CompanyLocationImport,

        /// <summary>
        /// 
        /// </summary>
        DocumentImport,

        /// <summary>
        /// 
        /// </summary>
        ExemptCertImport,

        /// <summary>
        /// 
        /// </summary>
        ItemImport,

        /// <summary>
        /// 
        /// </summary>
        SalesAuditExport,

        /// <summary>
        /// 
        /// </summary>
        SstpTestDeckImport,

        /// <summary>
        /// 
        /// </summary>
        TaxRuleImport,

        /// <summary>
        /// 
        /// </summary>
        TransactionImport,

        /// <summary>
        /// 
        /// </summary>
        UPCBulkImport,

        /// <summary>
        /// 
        /// </summary>
        UPCValidationImport,


    }
}

File: clients/php/enums/BatchType.php
<?php
/*
 * AvaTax Enum Class
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
namespace Avalara.AvaTax;

/**
 * @author Ted Spence <ted.spence@avalara.com>
 * @author Bob Maidens <bob.maidens@avalara.com>
 */class BatchType extends AvaTaxEnum 
{

    const AvaCertUpdate = "AvaCertUpdate";
    const AvaCertUpdateAll = "AvaCertUpdateAll";
    const BatchMaintenance = "BatchMaintenance";
    const CompanyLocationImport = "CompanyLocationImport";
    const DocumentImport = "DocumentImport";
    const ExemptCertImport = "ExemptCertImport";
    const ItemImport = "ItemImport";
    const SalesAuditExport = "SalesAuditExport";
    const SstpTestDeckImport = "SstpTestDeckImport";
    const TaxRuleImport = "TaxRuleImport";
    const TransactionImport = "TransactionImport";
    const UPCBulkImport = "UPCBulkImport";
    const UPCValidationImport = "UPCValidationImport";
}
